### PR TITLE
Fix use current selectable version instead of pipy version

### PR DESCRIPTION
**Pull request by tam0407**
For easier to test example code

### CODE REVIEW

File: selectable/templatetags/selectable_tags.py [hunk 1]
```diff
@@ -13,5 +13,5 @@ def include_jquery_libs(version='1.11.2', ui='1.11.3'):
 
 
 @register.inclusion_tag('selectable/jquery-css.html')
-def include_ui_theme(theme='base', version='1.11.3'):
+def include_ui_theme(theme='smoothness', version='1.11.3'):
```
Comment (mlavin) [inline selectable/templatetags/selectable_tags.py]: Why was this changed?

Comment (tam0407) [inline selectable/templatetags/selectable_tags.py]: Because http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/base/jquery-ui.css does not exists.

Comment (tam0407) [inline selectable/templatetags/selectable_tags.py]: The jquery-ui on the version on `pipy` is `1.8.23` and it comes with the `base` theme, but the selectable version from the project uses jquery-ui version `1.11.3`, which `base` theme doesn't available.

Comment (mlavin) [inline selectable/templatetags/selectable_tags.py]: :/ good catch then.
